IPython/terminal/ipapp.py

```diff
@@ -154,6 +154,7 @@ def make_report(self,traceback):
 
 aliases = dict(base_aliases)
 aliases.update(shell_aliases)
+aliases['term-title-format'] = 'TerminalInteractiveShell.term_title_format'
```

Comment:
You actually do not need to do this, as you did `.tag(config=True)` the CLI parser has autogenerated a flag called `--TerminalInteractiveShell.term_title_format`,  and I'm unsure we need a short for of this one. @takluyver @ellisonbg thoughts ?